### ISSUE TITLE
R CMD CHECK NOTE

### ISSUE THREAD

**Issue by pawelru** [insightsengineering/rbmi#373]
When doing R CMD CHECK of one of our internal systems I got following note. This seems to be easily fixable

```
* checking dependencies in R code ... NOTE
Namespaces in Imports field not imported from:
  ‘RcppParallel’ ‘rstantools’
  All declared Imports should be used.
```

**Comment by gowerc** [comment]
Hey @pawelru ,

Unfortunetly this isn't really possible to clean up without adverse effects. Both of these libraries are used as part of the installation process but neither of them are actually used in the source code itself.  i.e. I don't really want to import modules that the code doesn't use but likewise we can't drop them from the imports section of the DESCRIPTION file because they are needed.

In particular   [configure](https://github.com/insightsengineering/rbmi/blob/main/configure) uses rstantools to generate a templated makefile at installation time which contains a call to `RcppParallel`.

**Comment by pawelru** [comment]
Thanks for the explanation - I haven't got that deep into the codes. This ticket is just a heads-up from my side as we were doing R CMD CHECK for many packages including `rbmi` and I wanted to report discovered warnings / notes to the package maintainers.

You are definitely right about no clean way to address that. Did a quick research and it seems that `rstantools` suggests to  put a dummy imports into the codebase so as to silent this note [link](https://mc-stan.org/rstantools/articles/minimal-rstan-package.html#:~:text=TRUE%0A%23%27%20%40import%20methods-,%23%27%20%40import%20Rcpp,-%23%27%20%40importFrom%20rstan%20sampling).

**Comment by gowerc** [comment]
This has been addressed in #380
